README.md

```diff
@@ -32,26 +32,34 @@ React Frontend + GraphQL API   ┃┃
 
 # Getting started
 
-## Prerequisites
+Stack: **React**, **Hasura** graphql server & **vercel** serverless functions
 
-- NodeJS version 14
-- A backend API to connect this frontend app to. This app connects to the backend defined in REACT_APP_API_BASE_URL in the `.env` file. To setup the coordinape-backend app locally and use that as your API, [follow the instructions.](https://github.com/coordinape/coordinape-backend/blob/main/README.md)
-  - You can optionally use `https://staging-api.coordinape.com/api` if you don't want to run your API locally
-- An Infura project id: [Infura](https://infura.io)
-  - After you sign up for an account, go to Ethereum > Create New Project and the project ID will be available on the settings page
-- A browser with MetaMask installed (it's the officially supported wallet)
+### Prerequisites
+
+- NodeJS v14
+- Hasura cli version >= 2.1.1
+- Yarn
 - Docker
 
-## Getting Started (Frontend)
+## Quick Start
+
+- `yarn install`
+- Setup `.env` by copying from `.env.example`
+  - Set `LOCAL_WALLET_ADDRESS` for the wallet you want to use w/ seeding
+- `yarn setup` - Compile contracts and init git submodules
```

Comment:
the setup-hardhat.sh script is spelled wrong in this branch but is correct in `main`